src/QirRuntime/lib/QIR/intrinsicsMath.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <cmath>
+
+extern "C"
+{
+
+// Declarations:
+bool        quantum__qis__isnan__body(double d);
```

Comment:
The declarations are unnecessary here. Maybe, extract into a header file so
a) it's easier to find them (people often start exploration of a codebase from the headers)
b) when/if more complicated implementations are added, the header can be included into the files that test them

Reply:
Fixed.